`make` step fails when building with documentation

Hi! I'm seeing the following error message when trying to build `libva` with documentation,
```
Making all in doc
make[2]: Entering directory '/tmp/libva-20180919-53586-1g8vt3z/libva-2.2.0/doc'
make[2]: *** No rule to make target '../doc/va_footer.html', needed by 'Doxyfile'.  Stop.
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -30,6 +30,7 @@ if ENABLE_DOCS
 SUBDIRS += doc
 endif
 
+EXTRA_DIST=doc/va_footer.html
 # Extra clean files so that maintainer-clean removes *everything*
 MAINTAINERCLEANFILES = \
 	aclocal.m4 compile config.guess config.sub \

